src/quantum/HISTORY.rst

```diff
@@ -3,6 +3,12 @@
 Release History
 ===============
 
+0.12.0
+++++++
+* [2022-01-25] Version intended to work with QDK version v0.22.186636
+* Updated data plane generated client to API version 2021-11-01-preview.
+* Added support for displaying price estimate for a job when available.
```

Comment:
```suggestion
* Added support for displaying cost estimate for a job when available.
```

Reply:
Per discussion in email, let's call this as "cost estimate instead"

Reply:
Also, please update the string in [this line](https://github.com/IsraelMiles/azure-cli-extensions/blob/israelmiles/release/0.12.0/src/quantum/azext_quantum/commands.py#L46) to "Cost estimate", as below:

            `transformed_result['Cost estimate'] = price`